provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -311,6 +317,10 @@
           "type": "boolean",
           "description": "Skip the default dependency installation step - use this to customize the dependency installation (e.g. if using yarn or poetry)"
         },
+        "skipIntermediateDeployments": {
+          "type": "boolean",
+          "description": "Skip pending deployments (Consolidate multiple deployments of the same type into one deployment)"
```

Comment:
Maybe something like "Only run the latest deployment if multiple of the same type are queued"